modules/apps/27-interchain-accounts/keeper/relay.go

```diff
@@ -86,7 +97,9 @@ func (k Keeper) createOutgoingPacket(
 	return k.ComputeVirtualTxHash(packetData.Data, packet.Sequence), k.channelKeeper.SendPacket(ctx, channelCap, packet)
 }
 
-func (k Keeper) DeserializeTx(_ sdk.Context, txBytes []byte) ([]sdk.Msg, error) {
+// DeserializeCosmosTx will unmarshal and unpack a slice of transaction bytes
```

Comment:
Tiny nit: maybe drop the "will" and just use present tense `unmarshals and unpacks`

Feel free to leave until later

Reply:
addressed! https://github.com/cosmos/ibc-go/pull/457/commits/20bc33503c4ba5dd15d2abbe1c5f13fb5cdc77eb